en/what-do-i-need-to-publish/list-of-all-elements

```diff
@@ -0,0 +1,38 @@
+Full list of top-level elements in IATI
+=========
+
```

Comment:
Suggest adding some sort of introductory text to explain what the table is about, and what the columns mean.